x-pack/legacy/plugins/monitoring/public/components/metricbeat_migration/flyout/flyout.js

```diff
@@ -22,17 +22,19 @@ import {
   EuiText,
 } from '@elastic/eui';
 import { getInstructionSteps } from '../instruction_steps';
-import { Storage } from '../../../../../../../../src/legacy/ui/public/storage/storage';
+import { Storage } from '../../../../../../../../src/legacy/ui/public/storage';
```

Comment:
You can also do short hand paths here eg: `import { Storage } from 'ui/storage';`